### ISSUE TITLE
handle ambiguous/lower case nucleotides

### ISSUE THREAD

**Issue by cheberling** [HadrienG/InSilicoSeq#73]
InSilicoSeq throws errors when it comes across lower case versions of the four main nucleotides (actg), as well as the ambiguous nucleotides, such as Y, R, H, B.  I'm not sure, but it looks like N may be handled appropriately?

**Comment by HadrienG** [comment]
Hi!

Thanks for reporting this. I'll investigate further and fix in the next release.

Best,
Hadrien.

**Comment by HadrienG** [comment]
Hi again,

Could you share with me the error message you got? `iss generate` seems to work for me with IUPAC nucleotides.

**Comment by cheberling** [comment]
[iss_ambig_bp_errors_logs.tar.gz](https://github.com/HadrienG/InSilicoSeq/files/2586232/iss_ambig_bp_errors_logs.tar.gz)

See attached files.

**Comment by HadrienG** [comment]
```pytb
Traceback (most recent call last):
File "/usr/local/lib/python3.6/site-packages/joblib/externals/loky/process_executor.py", line 418, in _process_worker r = call_item()
File "/usr/local/lib/python3.6/site-packages/joblib/externals/loky/process_executor.py", line 272, in __call__ return self.fn(*self.args, **self.kwargs)
File "/usr/local/lib/python3.6/site-packages/joblib/_parallel_backends.py", line 567, in __call__ return self.func(*args, **kwargs)
File "/usr/local/lib/python3.6/site-packages/joblib/parallel.py", line 225, in __call__ for func, args, kwargs in self.items]
File "/usr/local/lib/python3.6/site-packages/joblib/parallel.py", line 225, in <listcomp> for func, args, kwargs in self.items]
File "/usr/local/lib/python3.6/site-packages/iss/generator.py", line 59, in reads forward, reverse = simulate_read(record, ErrorModel, i)
File "/usr/local/lib/python3.6/site-packages/iss/generator.py", line 134, in simulate_read forward.seq = ErrorModel.mut_sequence(forward, 'forward')
File "/usr/local/lib/python3.6/site-packages/iss/error_models/__init__.py", line 100, in mut_sequence nucl_choices[position][nucl][0], KeyError: 'a'
```

**Comment by HadrienG** [comment]
Right, so `1.3.1` fixed the error for indels but not for substitutions apparently

**Comment by HadrienG** [comment]
Should be fixed in `1.3.3`

**Comment by simroux** [comment]
Can confirm it fixed the issues I was having, thanks !
